### DISCUSSION TITLE
Bitlocker

### DISCUSSION THREAD

**Discussion by iisfaq** [ionuttbara/windows-defender-remover#24] [General]
In the following issue you wrote

https://github.com/jbara2002/windows-defender-remover/issues/22

The issue with bitlocker is known.

Known to you?, is there a list of these known issues anywhere?

Updating the readme.md would save others from the same issue, I was about to test the script and I do have bitlocker.

**Comment by ionuttbara** [comment]
i've said in the changelog of the version 12. Applying the remover script of version 12 , will remove bitlocker (will be unregistered from the system)

![image](https://user-images.githubusercontent.com/76656855/211511817-580126b5-2b55-4b55-b7f9-e090d0dd6290.png)

**Reply by ionuttbara** [reply]
Next time, will be a section of known issues in the readme.md.
